skfuzzy/fuzzymath/fuzzy_ops.py

```diff
@@ -661,6 +661,60 @@ def interp_membership(x, xmf, xx):
     return xxmf
 
 
+def interp_value(x, xmf, y):
+    """
+    Finds the degree of membership ``u(xx)`` for a given value of ``x = xx``.
```

Comment:
I think this needs a new one-liner. Perhaps something like "Find interpolated universe value(s) for a given fuzzy membership value."